modules/Bio/EnsEMBL/Analysis/Hive/Config/BestTargetted_subpipeline.pm

```diff
@@ -0,0 +1,902 @@
+
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'           => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+    'production_db_server'      => 'mysql-ens-meta-prod-1',
+    'production_db_port'        => '4483',
+
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps                => catdir($self->o('output_path'), 'genome_dumps'),
+    # This one is used by most analyses that run against a genome flatfile like exonerate, genblast etc. Has slice name style headers. Is softmasked
+    softmasked_genome_file      => catfile($self->o('genome_dumps'), $self->o('species_name').'_softmasked_toplevel.fa'),
+
+
+########################
+# db info
+########################
+
+    'cdna_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_cdna_'.$self->o('release_number'),
+      -host   => $self->o('cdna_db_server'),
+      -port   => $self->o('cdna_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    cdna2genome_db => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_cdna2genome_'.$self->o('release_number'),
+      -host   => $self->o('cdna2genome_db_server'),
+      -port   => $self->o('cdna2genome_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genewise_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genewise_'.$self->o('release_number'),
+      -host   => $self->o('genewise_db_server'),
+      -port   => $self->o('genewise_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'best_targeted_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_bt_'.$self->o('release_number'),
+      -host   => $self->o('genewise_db_server'),
+      -port   => $self->o('genewise_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+ 
+     'killlist_db' => {
+      -dbname => $self->o('killlist_db_name'),
+      -host   => $self->o('killlist_db_server'),
+      -port   => $self->o('killlist_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -driver => $self->o('hive_driver'),
+    },
+    
+
+#######################
+# Extra db settings
+########################
+    num_tokens => 10,
+
+
+  };
+}
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+
+}
+
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+  }
+}
```

Comment:
These can be removed as they are not really used in the pipeline

Reply:
done